packages/invoices/src/OutboundShipment/DetailView/tabs/GeneralTab.tsx

```diff
@@ -28,22 +28,18 @@ const Expand: FC = () => {
   );
 };
 
-export const GeneralTabComponent: FC<GeneralTabProps<OutboundShipmentRow>> = ({
-  data,
-  columns,
-}) => {
+export const GeneralTabComponent: FC<
+  GeneralTabProps<OutboundShipmentSummaryItem>
+> = ({ data, columns }) => {
   const { pagination } = usePagination();
-  const activeRows = data.filter(({ isDeleted }) => !isDeleted);
+  // const activeRows = data.filter(({ isDeleted }) => !isDeleted);
```

Comment:
this is looking like a placeholder, perhaps deleted lines are removed from `items`. Will wait and see!

Reply:
Was just rushing. 

We need to hide the summary rows when all batches are deleted, then the expando needs to hide batches when they're deleted, also.

Removing the item seems to make sense, though that would mean we put the lines elsewhere. Can go back to the `deletedLines` array, I suppose, but then the flag is redundant.

Could add an `isDeleted` to the summary item and do the same thing as we were doing. Seems to make the most sense to me 🤔